test/lazy/test_reuse_ir.py

```diff
@@ -104,15 +104,20 @@ def testAddSubFallback(self):
     def testBatchNorm(self):
         device = get_test_device()
         x = torch.randn(16, 3, 224, 224, device=device)
-        bn = torch.nn.BatchNorm2d(3).to(device=device)
+        weight = torch.randn(3, device=device)
+        bias = torch.randn(3, device=device)
+
         for i in range(10):
-            z = bn(x)
+            # BatchNorm2d does extra checks on dimensions which SymInts don't support yet
+            # so we call `torch.ops.aten.native_batch_norm` to bypass the checks.
+            z, _, _ = torch.ops.aten.native_batch_norm(x, weight, bias, None, None, True, 0.1, 1e-5)
```

Comment:
I don't see any use of symbolic ints here, how come we need to make this change?

Reply:
BatchNorm python module does multiplication and assertion and LTC doesn't have this part implemented yet :(